README.md

```diff
@@ -21,6 +21,7 @@ You can access the detailed rule description directly from your editor, using th
 
 Out of the box, SonarLint automatically checks your code against the following rules:
 
+- [Azure Resource Manager rules](https://rules.sonarsource.com/azureresourcemanager)
```

Comment:
We never added C# in this list; Maybe we can do it now? Hopefully we will soon get rid of this duplication in README and docs...